modules/hardware/common/kernel.nix

```diff
@@ -50,11 +50,21 @@ in {
           ++ config.ghaf.hardware.definition.gpu.pciDevices
           ++ config.ghaf.hardware.definition.audio.pciDevices
         ));
+        hugepagesz = 2; # MBytes
```

Comment:
if you use "optional scale suffix [kKmMgG]" here, the comment gets redundant and you can also change the line https://github.com/tiiuae/ghaf/pull/473/files#diff-7e202e79cee502470d29582cd4b072d36760f5cdc8c1eca7e87026577f04ae03R58 where you already use the scale suffix. Changing this also removes a source of error - if one wants to test different page sizes, they can't define the suffix on line 53 because it would break on line 58 with two suffices. Defining it only on line 53 solves the problem.

Reply:
Fixed